3.5/aql/graphs-kshortest-paths.md

```diff
@@ -176,48 +176,89 @@ train connections in Europe and North America.
     @endDocuBlock GRAPHKSP_01_create_graph
 {% endarangoshexample %}
 {% include arangoshexample.html id=examplevar script=script result=result %}
+
 Suppose we want to query a route from **Aberdeen** to **London**, and compare
 the outputs of SHORTEST_PATH and K_SHORTEST_PATHS with LIMIT 1. Note that while
 SHORTEST_PATH and K_SHORTEST_PATH with LIMIT 1 should return a path of the same
 length (or weight), they do not need to return the same path.
-{% arangoshexample examplevar="examplevar" script="script" result="result" %}
+
+{% aqlexample examplevar="examplevar" type="type" query="query" bind="bind" result="result" %}
+    @startDocuBlockInline GRAPHKSP_01_Aberdeen_to_London
+    @EXAMPLE_AQL{GRAPHKSP_01_Aberdeen_to_London}
+    @DATASET{kShortestPathsGraph}
+    FOR v, e IN OUTBOUND SHORTEST_PATH 'places/Aberdeen' TO 'places/London'
+    GRAPH 'kShortestPathsGraph'
+        RETURN [ v, e ]
```

Comment:
can we have a more compact reply without database specific items like `_id` etc.? 
maybe like this:  `RETURN {vertices: v[*].label, edges: e[*].label}`